### ISSUE TITLE
upload episode by link

### ISSUE THREAD

**Issue by rebelojose** [albertobeta/PodcastGenerator#29]
hi, any way to upload new episode by link with cloud service, without browse upload file to server?

normally i edit the .xml file manually. 
it is possible in future relases have option create new episode by link

regards
josé

**Comment by nkomarov** [comment]
YES, it would be really cool! Using a Google Drive public link instead of uploading to a hosting with a very limited space would be a great cost-saving point.

**Comment by nkomarov** [comment]
@emilengler Simply closing everything in a row? Really? Why?

**Comment by emilengler** [comment]
@nkomarov I explained this a lot in the past hours. This has the reason that PG3 gets released in a few hours and I simply don't want to and also can not give support for old versions. I hope this is understandable though :/

Sorry that I read these feature requests a bit to late, I joined the project in October this year so I'm not a long term PG user :/

I will add this to PG3

**Comment by emilengler** [comment]
Btw I just closed all issues because most of them were version dependent. As this one is a more general issue, I opened it again. Sorry if my behavior is a bit wrong. This is the first bigger project which I maintain and I also have Aspergers which means my social skills aren't always the best as well. :(
However I'm open for criticism so just tell me what's wrong :)

**Comment by nkomarov** [comment]
I can understand if you close an issue that is resolved in a new version. I mean, if a bug is fixed or a feature has been implemented. But even in this case, it makes sense to paste everywhere a simple comment "solved/implemented in the version to come very soon" before closing.

By silently closing all these issues without review you send a message that you just don't give a f*** to what people said.

**Comment by emilengler** [comment]
Ok, it wasn't meant at as this sorry

**Comment by emilengler** [comment]
Will work on this the next release or maybe even subrelease. Sorry but it is way too much work that I had expected

**Comment by 01Time** [comment]
I'm assuming this would address my issue as well. I have my mp3 files hosted at a different location that where PG is installed. So, I would love the option to be able to type the url of the specific mp3 instead of being forced to upload it.

**Comment by emilengler** [comment]
> I'm assuming this would address my issue as well. I have my mp3 files hosted at a different location that where PG is installed. So, I would love the option to be able to type the url of the specific mp3 instead of being forced to upload it.

Location in terms of server?

**Comment by 01Time** [comment]
`Location in terms of server?'

Yes. Where I installed PG has a relatively small file size limit and so I use another location (server) to store the media on, but it doesn't support PHP. At some point, it may be worth the money to put everything in one place, but in the meantime it would make it much easier to have the option to point to the url of the media location and have PG use that for the media url for the rss/xml file.

**Comment by pblommendaal** [comment]
+1

This would be very nice to have.

**Comment by DLSS604** [comment]
> YES, it would be really cool! Using a Google Drive public link instead of uploading to a hosting with a very limited space would be a great cost-saving point.

did this ever end up getting implemented ?
as i would also appreciate this ability.

**Comment by coldacid** [comment]
No, it hasn't, but it will be easy to add in when we make the switch to SQLite for tracking episode data in 4.0.
